src/publisher/api_v2_views.py

```diff
@@ -57,6 +58,14 @@ def is_authenticated(request):
     return val or False
 
 @api_view(['GET'])
+@renderer_classes((StaticHTMLRenderer,))
+def ping(request):
+    "returns a test response"
+    # produce standard authenticated header
+    is_authenticated(request)
+    return Response('pong', content_type='text/plain; charset=UTF-8', headers={'Cache-Control': 'must-revalidate, no-cache, no-store, private'})
```

Comment:
if these cache headers are different for the /ping endpoint than for all the others, then this is fine, otherwise, caching headers are handled in custom middleware in `core.middleware` (beneath the kong authentication middleware) and this can be removed.